### PR TITLE
Fix feedback issues

### LINKED ISSUE #3552: feedback: Database error codes | Tarantool
**Product:** Tarantool
**Root document:** https://www.tarantool.io/en/doc/latest/reference/reference_lua/errcodes/
New link: https://github.com/tarantool/tarantool/blob/master/src/box/error.h

# Details

> <…>e popular codes. A complete list of errors can be found in file
> **|errcode.h|** in the source tree.

https://www.tarantool.io/en/doc/latest/reference/reference_lua/errcodes/

link to a 404

### LINKED ISSUE #3559: feedback: Automated leader election | Tarantool
**Related dev. issue(s):** GitHub link(s)

**Product:** Tarantool
**Root document:** https://www.tarantool.io/en/doc/latest/concepts/replication/repl_leader_elect/#leader-election-and-synchronous-replication
**SME:** @ sergepetrenko 

# Details

> <…>election process
> 
> 
> 
> Leader election and synchronous replication
> **|Leader election and synchronous replication are implemented in Tarantool as
> a modification of the Raft
> algorithm.
> Raft is an algorithm of synchronous replication and automatic leader election.
> Its complete description can be found in the corresponding document.|**
> In Tarantool, synchronous replication and leader election
> are s<…>

https://www.tarantool.io/en/doc/latest/concepts/replication/repl_leader_elect/#leader-election-and-synchronous-replication

This place lacks an explanation on how to reach "canonical Raft" experience:
* The user should only have synchronous spaces
* The user should have the default `replication_synchro_quorum` setting = `'N / 2 + 1'`
* `replication_synchro_timeout` should be set to infinity.
* `election_fencing_mode` should be default - `'soft'`. `'strict'` is also acceptable, but it's too restrictive

It's important to mention that with such settings the system behaves exactly in accordance with Raft.

Requested by @sergepetrenko

### PR DESCRIPTION
**Pull request by xuniq**
* Fixes #3552 - fix broken link
* Fixes #3559 - add explanation on how to reach "canonical Raft" experience
* Fix translation and wording

Deploy to dev: https://docs.d.tarantool.io/ru/doc/feedback-issues/concepts/replication/repl_leader_elect/

### CODE REVIEW

**Review by sergepetrenko — APPROVED** [review]
Thanks!

File: doc/concepts/replication/repl_leader_elect.rst [hunk 1]
```diff
@@ -39,6 +39,16 @@ in the cluster but don't use synchronous spaces at all.
 Synchronous replication has a separate :ref:`documentation section <repl_sync>`.
 Leader election is described below.
 
+..  note::
+
+    The system behavior can be specified exactly according to the Raft algorithm. To do this:
+
+    *   Ensure that the user only has synchronous spaces.
```
Comment (andreyaksenov) [inline doc/concepts/replication/repl_leader_elect.rst]: maybe, 'user has only...' would be clearer (if `only` refers to synchronous spaces)

File: doc/concepts/replication/repl_leader_elect.rst [hunk 2]
```diff
@@ -39,6 +39,16 @@ in the cluster but don't use synchronous spaces at all.
...
+    *   Set the :ref:`replication_synchro_quorum <repl_leader_elect_config>` option to ``N / 2 + 1``.
+    *   Set the :ref:`replication_synchro_timeout <cfg_replication-replication_synchro_timeout>` setting to infinity.
```
Comment (andreyaksenov) [inline doc/concepts/replication/repl_leader_elect.rst]:
```suggestion
    *   Set the :ref:`replication_synchro_timeout <cfg_replication-replication_synchro_timeout>` option to infinity.
```

File: doc/concepts/replication/repl_leader_elect.rst [hunk 3]
```diff
@@ -39,6 +39,16 @@ in the cluster but don't use synchronous spaces at all.
...
+    *   In the :ref:`election_fencing_mode <repl_leader_elect_config>` option, select either ``soft`` mode (the default)
+        or ``strict`` mode, which is more restrictive.
```
Comment (andreyaksenov) [inline doc/concepts/replication/repl_leader_elect.rst]:
```suggestion
    *   In the :ref:`election_fencing_mode <repl_leader_elect_config>` option, select either the ``soft`` mode (the default)
        or the ``strict`` mode, which is more restrictive.
```

**Review by andreyaksenov — COMMENTED** [review]
just some minor suggestions

**Pull request merged**